temporaltest/server.go

```diff
@@ -38,8 +38,8 @@ import (
 	"go.temporal.io/sdk/worker"
 	"go.temporal.io/server/common/dynamicconfig"
 	"go.temporal.io/server/common/log"
-	"go.temporal.io/server/internal/temporalite"
 	"go.temporal.io/server/temporal"
+	temporalite "go.temporal.io/temporaltest/internal"
```

Comment:
NOTE: I moved this package in here under `internal` as it's only used here and I was told once we might want to refactor this in a major way. Therefore, I'm not exposing that particular package.